projects/m1/015-is-it-a-leap-year/js/index.js

```diff
@@ -0,0 +1,6 @@
+let year = prompt("Chose an year")
```

Comment:
attenzione il `prompt` non restituisce un numero, ma sempre una stringa. Tu tratti subito `year` come un numero ma non lo è, questo può generare comportamenti inattesi, devi sempre convertire gli input in numeri in maniera esplicita, studiati i metodi `parseInt` e `parseFloat`